### PR TITLE
feat: input with suffix

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Dependant of: https://github.com/ArkEcosystem/marketsquare.io/pull/2201

1. Adds and `input-with-suffix` component (used on the claim form on msq)
2. Updates the CSS to properly show the suffix control even when it has an error
3. To test, use the new claim form on the msq PR and see the input for the user email

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/UserInterfaceServiceProvider.php [hunk 1]
```diff
@@ -175,7 +175,7 @@ private function registerBladeComponents(): void
         Blade::component('ark::inputs.date-picker', 'ark-date-picker');
         Blade::component('ark::inputs.input', 'ark-input');
         Blade::component('ark::inputs.input-with-icon', 'ark-input-with-icon');
-        Blade::component('ark::inputs.input-with-prefix', 'ark-input-with-prefix');
+        Blade::component('ark::inputs.input-with-control', 'ark-input-with-control');
```
Comment (ItsANameToo) [inline src/UserInterfaceServiceProvider.php:178]: can this be changed to two components:
- `input-with-prefix`
- `input-with-suffix`

you can then have those two components as wrappers that both call the `input-with-control` component but with a predefined property for the location of it. Reason being is that:
1. it avoids a breaking change as the existing prefix component will still exist
2. it makes it clearer (to me at least) what the component does, as `input-with-control` feels ambiguous to me (what's a control?)

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: src/UserInterfaceServiceProvider.php (hunk 1 above)
Comment (alfonsobries) [reply src/UserInterfaceServiceProvider.php]: @ItsANameToo done

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

File: resources/assets/css/_forms.css [hunk 2]
```diff
@@ -42,16 +42,17 @@
     @apply text-theme-primary-600;
 }
 
-.input-group:focus-within .input-wrapper-with-prefix,
+.input-group:focus-within .input-wrapper-with-prefix-suffix,
```
Comment (ItsANameToo) [inline resources/assets/css/_forms.css:45]: these now have `-prefix-suffix` but shouldn't that be just `-suffix`? so you get `input-wrapper-with-prefix` and `input-wrapper-with-suffix`

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**